Tasks/UseNodeV1/usenode.ts

```diff
@@ -22,7 +22,6 @@ async function run() {
         if (version) {
             const checkLatest: boolean = taskLib.getBoolInput('checkLatest', false);
 
-            // TODO: installer doesn't support proxy
```

Comment:
Since I've implemented this TODO item, I am just removing the comment to avoid confusion